android/src/main/java/com/mattermost/networkclient/NetworkClient.kt

```diff
@@ -61,6 +83,20 @@ class NetworkClient(private val baseUrl: HttpUrl? = null, private val options: R
         }
     }
 
+    fun importClientP12AndRebuildClient(p12FilePath: String, password: String) {
+        importClientP12(p12FilePath, password)
```

Comment:
by reading `buildHandshakeCertificates(options)` it seems that `importClientP12` can throw, don't you need to handle that here?